sonar-plugin-api/src/main/java/org/sonar/api/server/ws/WebService.java

```diff
@@ -116,6 +119,10 @@ public NewController createController(String path) {
       return new NewController(this, path);
     }
 
+    /**
+     * @deprecated since 6.5 use {@link #addController(Controller)}
+     */
+    @Deprecated
     private void register(NewController newController) {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)